bittensor/cli.py

```diff
@@ -16,6 +16,7 @@
 # DEALINGS IN THE SOFTWARE.
 
 import sys
+import shtab
```

Comment:
Nicely Done! Lets also up the minor version for this release then, since this will introduce a new requirement